src/xrGame/quadtree_inline.h

```diff
@@ -102,6 +108,9 @@ IC void CSQuadTree::insert(_object_type* object)
         if (!*node)
             *node = m_nodes->get_object();
 
+        if (!node)
+            return;
```

Comment:
[Back to the old discussion](https://github.com/OpenXRay/xray-16/pull/1432#discussion_r1306691967), this check doesn't make sense, since we already dereferenced the `node` pointer two lines above.